src/libsync/checksums.cpp

```diff
@@ -17,6 +17,7 @@
 #include "syncfileitem.h"
 #include "propagatorjobs.h"
 #include "account.h"
+#include "configfile.h"
```

Comment:
Since if it's for a debugging purpose, maybe an environment variable would work.  But I think we could also leave it out. Who is this debug option for?  Developers can as well comment the code that does the checking.